### PR TITLE
Add time monitoring

### PR DESCRIPTION
**Pull request by ohiliazov**
Add time monitoring feature

### CODE REVIEW

File: sgftools/utils.py [hunk 1]
```diff
@@ -102,3 +105,78 @@ def join_list_into_str(list_to_join, separator):
     :return: string
     """
     return separator.join(map(str, list_to_join))
+
+
+def winrate_transformer(stdev, verbosity):
+    """
+    Stretch winrate out the middle range and squashe the extreme ranges
```
Comment (aTan-aka-Xellos) [inline sgftools/utils.py]: if you know how it works already - better add detailed explanation :)

File: sgftools/leela.py [hunk 2]
```diff
@@ -224,6 +224,18 @@ def reset(self):
         """
         self.send_command('clear_board')
 
+    def save_sgf(self, sgf_fn):
+        """
+        Save sgf to checkpoint file
```
Comment (aTan-aka-Xellos) [inline sgftools/leela.py]: not to checkpoint, but to given filename

File: sgftools/leela.py [hunk 3]
```diff
@@ -224,6 +224,18 @@ def reset(self):
...
+        """
+        self.send_command('printsgf %s' % sgf_fn)
+
+    def load_sgf(self, sgf_fn):
+        """
+        Load sgf from checkpoint file
```
Comment (aTan-aka-Xellos) [inline sgftools/leela.py]: load from given file

File: sgftools/leela.py [hunk 4]
```diff
@@ -224,6 +224,18 @@ def reset(self):
         """
         self.send_command('clear_board')
 
+    def save_sgf(self, sgf_fn):
+        """
+        Save sgf to checkpoint file
+        """
+        self.send_command('printsgf %s' % sgf_fn)
+
+    def load_sgf(self, sgf_fn):
```
Comment (aTan-aka-Xellos) [inline sgftools/leela.py]: right now it's usage is not implemented yet?

Comment (ohiliazov) [reply sgftools/leela.py]: Nope

**Pull request merged**